pom.xml

```diff
@@ -25,12 +25,6 @@
 
   <dependencyManagement>
     <dependencies>
-      <dependency>
-        <groupId>org.keycloak</groupId>
-        <artifactId>keycloak-core</artifactId>
-        <version>${keycloak.version}</version>
-        <scope>provided</scope>
-      </dependency>
```

Comment:
because this is marked as "provided" here, we were missing it from the keycloak-config image when we use maven to add the project dependencies.
its already sucked in from other dependencies, so probably no need to explicitly manage its version here.